tests/vmi_hook_sidecar_test.go

```diff
@@ -77,7 +78,7 @@ var _ = Describe("[sig-compute]HookSidecars", decorators.SigCompute, func() {
 	BeforeEach(func() {
 		virtClient = kubevirt.Client()
 
-		vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+		vmi = libvmifact.NewAlpine(libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()))
```

Comment:
I'm not really sure what does it mean to have an interface without a network.
I would expect this to fail.